### DISCUSSION TITLE
🌐 Website?

### DISCUSSION THREAD

**Discussion by rahulduhan** [Psyhackological/AAA#51] [Ideas]
It is really hard to navigate a GitHub `README`.
It would be great if there was a website, 
Using display `flex` more apps count displayed at once, and it would look great and work great.
Tags could be added, which should help in easy navigation.
I am learning HTML and CSS and want to test my skills with making a website. It would be a great knowledgeable project for me to work on.
I used this[ website](https://winstall.app/) in my old windows day. We could take some inspiration from here...

**Comment by Psyhackological** [comment]
### Sorry for late comment. I'm not gonna lie I did not know this project of mine would that explode.
> It is really hard to navigate a GitHub README.

You're right, the only reason that I make this in MarkDown is easy to setup and add + no issues with web-development.

> It would be great if there was a website,

I agree.

> Using display flex more apps count displayed at once, and it would look great and work great.

Yeah sure, however I would be up for [Tailwind CSS](https://tailwindcss.com/). I've used to work with bootstrap, but in my opinion is it too bloated and I want to give a try something new like Tailwind.

> Tags could be added, which should help in easy navigation.

Yeah I know 2 websites that have GitHub's repo, however they have website generated through this repo.
1. https://quickref.me/
	[GitHub link](https://github.com/Randy8080/reference)
2. https://learnxinyminutes.com/
	[GitHub link](https://github.com/adambard/learnxinyminutes-docs)


We could do the same. They use `.html.markdown`, `markdown` or normal `md` extensions.

> I am learning HTML and CSS and want to test my skills with making a website. It would be a great knowledgeable project for me to work on.

You can try, I'll be honest I am not good with web-development. But if you really want to try making it make it very simple and minimalist.

> I used this[ website](https://winstall.app/) in my old windows day. We could take some inspiration from here...

Sure, that looks really good. I also like [PrivacyGuides design](https://www.privacyguides.org/tools/).

**Reply by rahulduhan** [reply]
>> You can try, I'll be honest I am not good with web-development. But if you really want to try making it make it very simple and minimalist.

I would, but my college classes are going to start from tomorrow and I would rarely get free time to learn more of Web Dev. 
Although I would have almost no time, I would do my best to contribute, I just might be super slow and inconsistent, but I will make a webpage out of all the contributions for sure.

**Reply by Psyhackological** [reply]
Do whatever you need to do. This is a community project which means everything is optional, but if you have spare time (or money but not on this project) then spend it on it.
There is no haste. I have some project to finish and I'll be honest again - website to this project is not my priority right now. Maybe someday.

**Comment by Psyhackological** [comment]
Closing old discussion. If anyone wants to create a website dedicated from and to this list, open a new Issue or Discussion.
